spark/src/main/java/org/apache/iceberg/spark/actions/BaseSnapshotUpdateSparkAction.java

```diff
@@ -17,9 +17,10 @@
  * under the License.
  */
 
-package org.apache.iceberg.actions;
+package org.apache.iceberg.spark.actions;
```

Comment:
These changes are unrelated as well, but not a problem if you want them in this PR imho

Reply:
Fixed.